lib/ansible/modules/cloud/vmware/vmware_local_role_facts.py

```diff
@@ -53,8 +53,8 @@
     description: Facts about role present on ESXi host
     returned: always
     type: dict
-    sample: {
-        "AnsiUser1": {
```

Comment:
Add a note in notes section stating that this returns list of dict from version 2.8

Reply:
done. thx.